docs/feature_mouse_keys.md

```diff
@@ -42,6 +42,7 @@ In your keymap you can use the following keycodes to map key presses to mouse ac
 Mouse keys supports two different modes to move the cursor:
 
 * **Accelerated (default):** Holding movement keys accelerates the cursor until it reaches its maximum speed.
+* **Kinetic:** Holding movement keys accelerates the cursor following a quadratic curve until it reaches its maximum speed.
```

Comment:
```suggestion
* **Kinetic:** Holding movement keys accelerates the cursor with its speed following a quadratic curve until it reaches its maximum speed.
```

This could have been interpreted as being synonymous to the Accelerated mode, since the current phrase suggests that the quadratic curve is applied to the position [implying constant acceleration].